README.md

```diff
@@ -1 +1,4 @@
-# musicrs
\ No newline at end of file
+# musicrs
+
+### Audio Trimmer 
```

Comment:
Lets keep it in References heading since your module is the one that we are using